src/components/NcPopover/NcPopover.vue

```diff
@@ -385,22 +386,27 @@ export default {
 			await this.useFocusTrap()
 			this.addEscapeStopPropagation()
 
+			setTimeout(() => {
```

Comment:
Instead of reading a CSS variable and adding a new task with timeout, can we use a native event here?

```js
this.getPopoverContentElement().addEventListener('transitionend', () => {
	this.$emit('after-show')
}, { once: true })
```

Reply:
Love it yeah! Didn't realized `transitionend` was an option

Reply:
> Instead of reading a CSS variable and adding a new task with timeout, can we use a native event here?
> 
> ```js
> this.getPopoverContentElement().addEventListener('transitionend', () => {
> 	this.$emit('after-show')
> }, { once: true })
> ```

Was testing this, let me push a followup if it works :hourglass: